### DISCUSSION TITLE
Does anyone know how to how to turn all of this heart-shaped food real using ControlNet Models?

### DISCUSSION THREAD

**Discussion by Njasa2k** [lllyasviel/ControlNet#83] [Q&A] [answered]
A redditor asked the Stable Diffusion community to turn this sketch real, I thought this would be the perfect job for ControlNet but I'm not sure which model would be the best for this task.
![m5jyih5bsmia1](https://user-images.githubusercontent.com/99513716/219522117-5ded0b1a-6fce-4aef-a4be-34e7aeb84c79.png)
Somebody tried the scribble model but it mismanaged about a few of them.
![f8bkw4elmoia1](https://user-images.githubusercontent.com/99513716/219522684-b4aea341-f0f9-43b6-a8c3-7900bdeb0af1.png)

Can anyone know which ControlNet model would succeed in this task? Segmentation Mapping?

**Comment by geroldmeisinger** [comment, marked as answer]
you might look into "regional prompting", which is a different concept
